lib/matplotlib/font_manager.py

```diff
@@ -1099,15 +1099,18 @@ def score_family(self, families, family2):
         Returns a match score between the list of font families in
         *families* and the font family name *family2*.
 
-        An exact match anywhere in the list returns 0.0.
+        An exact match at the head of the list returns 0.0.
 
-        A match by generic font name will return 0.1.
+        A match further down the list will return between 0 and 1.
 
         No match will return 1.0.
         """
         if not isinstance(families, (list, tuple)):
             families = [families]
+        elif len(families) == 0:
```

Comment:
Now checking for empty families list here. The unchecked division by len(families) used to be inside the loop, where it was not reachable if len(families) == 0. Thanks for catching that!